CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
```

Comment:
Maybe you can include the commands needed to clone the repo, create a branch, build _etc._? Have a look at https://github.com/Parquery/revproxyry#development, it wasn't clear to me how to do that the first time.